### PR TITLE
Fix SceneViewer intent params

### PR DESCRIPTION
**Pull request by elalish**
Fixes #2093 and some related problems. Also cleans up the code since we don't have to worry about IE11 support anymore.

### CODE REVIEW

**Comment by elalish** [comment]
FYI @tpsiaki

File: packages/model-viewer/src/features/ar.ts [hunk 1]
```diff
@@ -268,31 +268,37 @@ configuration or device capabilities');
...
+        params.set('enable_vertical_placement', 'true');
+      }
+      if (params.has('sound')) {
+        const soundUrl = new URL(params.get('sound')!, location);
+        params.set('sound', soundUrl.toString());
+      }
+      if (params.has('link')) {
+        const linkUrl = new URL(params.get('link')!, location);
+        params.set('link', linkUrl.toString());
       }
```
Comment (tpsiaki) [inline packages/model-viewer/src/features/ar.ts:297]: it seems not the best that users pass a model url w/ sound & link get params that are used for the actual model request in addition to sending to Scene Viewer for sound/link. I don't think there's a great alternative solution though without adding Scene Viewer specific model-viewer attributes which might be more trouble than they're worth.

File: packages/model-viewer/src/features/ar.ts [hunk 2]
```diff
@@ -268,31 +268,37 @@ configuration or device capabilities');
      * the current device.
      */
     [$openSceneViewer]() {
-      // This is necessary because the original URL might have query
-      // parameters. Since we're appending the whole URL as query parameter,
-      // ? needs to be turned into & to not lose any of them.
-      const gltfSrc = this.src!.replace('?', '&');
```
Comment (tpsiaki) [inline packages/model-viewer/src/features/ar.ts:274]: removing this makes me happy :)

**Review by tpsiaki — APPROVED** [review]

File: packages/model-viewer/src/features/ar.ts (hunk 1 above)
Comment (elalish) [reply packages/model-viewer/src/features/ar.ts:297]: Agreed, it's not the best, but no one has complained so far. At least it's fairly unlikely their server is looking for the same query params as SceneViewer, but an intersection would be a problem.

**Pull request merged**